webpack/loader-set-webpack-public-path/package.json

```diff
@@ -12,13 +12,11 @@
     "test": "gulp test"
   },
   "dependencies": {
+    "@microsoft/set-webpack-public-path-plugin": "~0.0.0",
```

Comment:
It should get bumped to `~1.0.0` upon publish, correct?

---
In reply to: [130456033](https://github.com/Microsoft/web-build-tools/pull/288#discussion_r130456033) [](ancestors = 130456033)